### DISCUSSION TITLE
zoomblur with mask issue

### DISCUSSION THREAD

**Discussion by MarcVdH** [cables-gl/cables_docs#679] [Answers & Questions] [answered]
In several WIP patches i use zoom-blur near the edges of the canvas, trying to keep the center 100% sharp using a mask.
I tried all kinds of masks in all kind of formats but always ended up with a clearly visible transition between the sharp center and the blurred border area.
finally after experimenting forever I found that by reducing the strenght of the mask one can get rid of the visible transition, however this also slightly blurs the center area of the canvas, which is a no go for what I need.
Is it possible to keep an area 100% sharp and have a smooth transition towards a zoom-blurred outside?

my zoomblur test patch:
https://cables.gl/p/SdTEq3
![zoom blur transition issue](https://github.com/cables-gl/cables_docs/assets/126583906/20e9f821-2002-4f76-879e-91f594260e32)

**Comment by MarcVdH** [comment]
an example patch that illustrates the issue hopefully better
https://cables.gl/p/aDPhln
to get rid of the visible ellipse, I need to lower the  "amount" for the mask texture  to .88 which is a clear no go

**Comment by pandrr** [comment, marked as answer]
thx for reporting, adjusted the shader, please try again on dev.cables.gl
